storage/tools/dump_tree/dumplib/dumplib.go

```diff
@@ -203,12 +202,18 @@ func createTree(as storage.AdminStorage) (*trillian.Tree, crypto.Signer) {
 	return t, cSigner
 }
 
+// Args are the commandline arguments one can pass to Main
```

Comment:
Can we call this Options, not Args as the latter sounds like it interacts with argv[].